Kinvey/KinveyTests/NetworkStoreTests.swift

```diff
@@ -3518,6 +3516,14 @@ class NetworkStoreTests: StoreTestCase {
             switch result {
             case .success(let products):
                 XCTAssertEqual(products.count, 21000)
+                XCTAssertNotNil(products.first)
+                if let product = products.first {
+                    XCTAssertEqual(product.entityId, "1")
+                }
+                XCTAssertNotNil(products.last)
+                if let product = products.last {
+                    XCTAssertEqual(product.entityId, "21000")
```

Comment:
Out of curiosity, why is the `entityId` value here `21000`, and not `21001`?

Reply:
the range is starting from 0 to 20999, so that's why i'm doing `+ 1` in my mock response